nss-cohort-36/bangazon-workforce-mgt-virtual-magicians
# Sp training details

# Description
Create New Training function is complete at `http://localhost:8000/training/form`
Closes https://github.com/nss-cohort-36/bangazon-workforce-mgt-virtual-magicians/issues/31
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. Ensure that `connection.py` contains the appropriate absolute path for your machine to test locally
1. `git checkout master`
1. `git fetch --all`
1. `git checkout sp-training-details`
1. [Enter specific testing instructions here to approve]
# Checklist:
